exercises/triangle/bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "triangle",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-prelude": "^2.1.0",
+    "purescript-console": "^2.0.0",
+    "purescript-either": "~2.0.0",
```

Comment:
I'm not familiar with bower. I added these with `bower install --save purescript-whatever`. Should I do then and then edit the `bower.json` file, or is there a way I can get bower to use `^`?